### DISCUSSION TITLE
Dev Blog: 2023 Year in Review

### DISCUSSION THREAD

**Discussion by justintadlock** [WordPress/developer-blog-content#176] [Topic Ideas]
@bph and I have discussed publishing a "year in review" type of post for the Developer Blog. The timeline would be to publish this early January 2024.

I haven't had a chance to outline what this might look like yet, but the primary goal would be to share everything that was published on the blog in the past year. It might be a good idea to mention a few of the bigger WordPress highlights too.

For reference, here are a few examples that I wrote while working for WP Tavern. The end result, of course, would be a bit different for the Dev Blog, but I'm hoping this might give us a few ideas:

- https://wptavern.com/reflecting-on-2019-the-year-in-review
- https://wptavern.com/2020-step-away-from-the-fire
- https://wptavern.com/wp-taverns-2021-year-in-review

Ideas welcome.

**Comment by bph** [comment]
I love the idea, @justintadlock and I'd be happy to collaborate with you, and provide data, like: "As per December 14, 2023, the Developer Blog has 40 contributors, of which 24 were new." 🙌

Also, other milestones could be the most read post, the best day of traffic. most commented post... 
Still thinking....

**Reply by justintadlock** [reply]
Yes, any and all stats we can share would be great.

**Comment by abhansnuk** [comment]
I like this idea and think if we can add unique aspects to it too, it will help the Marketing Team with promoting this as a great resource. Happy to help with it too. Thanks Justin.

**Comment by bph** [comment]
Discussion is moved to the issue after approval. Please continue the conversation there 😃 

- https://github.com/WordPress/developer-blog-content/issues/187
